guides/store-retrieve-messages.md

```diff
@@ -0,0 +1,162 @@
+# Retrieve Messages Using Waku Store
```

Comment:
Should note that store is best-effort and doesn't replace someting like e2e data sync

Also worth mentioning the most common way of using store protocol, which is to do it when you start dapp, e.g. go to browser etc, then do it once and after that use relay/filter

Reply:
What do you mean by "e2e data sync"?

Are you saying that we should not provide guarantees on the fact that a dApp would be able to retrieve all messages via store?

Meaning that _e2e data sync_ is a way to ensure that no message is ommited.

We do not provide any protocol for _e2e data sync_ yet, do we? We only have FT store but that's more to improve resilience, not provide a guarantee of data availability, right?

Reply:
Store is best-effort, it doesn't deal with end to end consistency

See MVDS and Status specs for example